examples/rtfm-demo/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "rtfm-demo"
+version = "0.1.0"
+authors = ["James Munns <james.munns@ferrous-systems.com>"]
+edition = "2018"
+
+[dependencies]
+cortex-m-rtfm = "0.4"
```

Comment:
I've been using `"~0.4"` rather than an exact version for these in other personal projects... hasn't caused me any problems yet(!)   Do we have a preference for one over the other?

Reply:
`0.4` and `~0.4` are identical. It's `=0.4` that's different (exact version requirement).

We should have a clear preference for `0.4`/`~0.4`, in my opinion. Exact version requirements will sooner or later lead to trouble in someone's build, due to mismatched version requirements.